src/components/Table/Table.jsx

```diff
@@ -72,13 +86,27 @@ export function Table({
         dataCy={dataCy}
         {...rest}
       >
+        {withColumnChooser ? (
+          <HeaderUI>
```

Comment:
Is it possible to tweak this so that it can handle the MVP goal of having a header that shows the view name but not column sorting? [Relevant spec.](https://paper.dropbox.com/doc/Spec-List-View--BRCgjs8z4tVOOsjbO2y_dKhDAg-H9rEIGV717YlfL8JFCxoW#:uid=700259760955942417510932&h2=Folder-Header) Feels like we'd need to check for `withHeader` and `withColumnChooser` but for MVP only `withHeader` would be true.

If it's helpful, in my current branch I use an `H1` with the following CSS:
```
  h1 {
    height: 60px;
    font-size: 18px;
    font-weight: 400;
    line-height: 23.04px;
    padding-top: 16px;
    padding-bottom: 19px;
    padding-left: 15px;
    color: ${getColor('charcoal.800')};
  }
```

Reply:
Done (use `headerContent` and/or `withColumnChooser`)